internal/js/modules/k6/browser/tests/get_by_test.go

```diff
@@ -594,6 +592,42 @@ func TestGetByRoleSuccess(t *testing.T) {
 				}
 			})
 		}
+
+		// We test the 'document' role independently, because the expectations
```

Comment:
Dear reviewer, take a look at the following discussion https://github.com/grafana/k6/pull/5106#discussion_r2314177449 to get more context why this changed is needed. Thanks! 🙇🏻